account_operating_unit/tests/test_account_operating_unit.py

```diff
@@ -0,0 +1,157 @@
+                'company_id': self.company.id,
+                'company_ids': [(4, self.company.id)],
+                'operating_unit_ids': [(4, self.b2c.id)],
+                'groups_id': [(6, 0, [self.grp_acc_manager.id])]
+            })
+
+        # Create a cash account 1
+        user_type = self.env.ref('account.data_account_type_liquidity')
+        self.cash1_account_id = self.account_model.create({
+            'name': 'Cash 1 - Test',
+            'code': 'test_cash_1',
+            'user_type_id': user_type.id,
+            'company_id': self.company.id,
+        })
+
+        # Create a journal for cash account 1, associated to the main
+        # operating unit
+        self.cash_journal_ou1 = self.journal_model.create({
+            'name': 'Cash Journal 1 - Test',
+            'code': 'test_cash_1',
+            'type': 'cash',
+            'company_id': self.company.id,
+            'default_debit_account_id': self.cash1_account_id.id,
+            'default_credit_account_id': self.cash1_account_id.id,
+            'operating_unit_id': self.ou1.id
+        })
+
+        # Create a cash account 2
+        user_type = self.env.ref('account.data_account_type_liquidity')
+        self.cash2_account_id = self.account_model.create({
+            'name': 'Cash 2 - Test',
+            'code': 'test_cash_2',
+            'user_type_id': user_type.id,
+            'company_id': self.company.id,
+        })
+
+        # Create a journal for cash account 2, associated to the operating
+        # unit B2B
+        self.cash2_journal_b2b = self.journal_model.create({
+            'name': 'Cash Journal 2 - Test',
+            'code': 'test_cash_2',
+            'type': 'cash',
+            'company_id': self.company.id,
+            'default_debit_account_id': self.cash2_account_id.id,
+            'default_credit_account_id': self.cash2_account_id.id,
+            'operating_unit_id': self.b2b.id
+        })
+
+    def _prepare_invoice(self, operating_unit_id):
+        line_products = [(self.product1, 1000),
+                         (self.product2, 500),
+                         (self.product3, 800)]
+        # Prepare invoice lines
+        lines = []
+        acc_type = self.env.ref('account.data_account_type_expenses')
+        for product, qty in line_products:
+            line_values = {
+                'name': product.name,
+                'product_id': product.id,
+                'quantity': qty,
+                'price_unit': 50,
+                'account_id': self.env['account.account'].search(
+                    [('user_type_id', '=', acc_type.id)], limit=1).id
+            }
+            lines.append((0, 0, line_values))
+        inv_vals = {
+            'partner_id': self.partner1.id,
+            'account_id': self.partner1.property_account_payable_id.id,
+            'operating_unit_id': operating_unit_id,
+            'name': "Test Supplier Invoice",
+            'reference_type': "none",
```

Comment:
@bjeficent @AaronHForgeFlow  No need to `reference_type` in v12

Reply:
Thanks for the notice. Could you create a PR with the fix?